ProjectMarsLanguageFeature/Pages/LanguagesPage.cs

```diff
@@ -20,7 +20,7 @@ public class LanguagesPage
         //Locate Add new button and click
         public void AddLanguage(IWebDriver driver)
         {
-            locateCickObj.LocateClickAddNew(driver); 
+            locateCickObj.LocateClickAddNew(driver);
```

Comment:
can you not pass the driver around to action methods instead either pass the driver via the constructor or extend to common driver

Reply:
Sure